docs/data/material/guides/routing/LinkRouterWithTheme.tsx

```diff
@@ -33,7 +33,7 @@ const theme = createTheme({
     MuiLink: {
       defaultProps: {
         component: LinkBehavior,
-      } as LinkProps,
+      },
```

Comment:
Could you also remove the casting from the code snippet which is shown in https://mui.com/material-ui/guides/routing/#global-theme-link which is in `routing.md`?

Reply:
done https://deploy-preview-35924--material-ui.netlify.app/material-ui/guides/routing/#global-theme-link